cmd/mt-store-cat/main.go

```diff
@@ -186,7 +188,7 @@ func main() {
 		fmt.Println("Retrieves timeseries data from the cassandra store. Either raw or with minimal processing")
 		fmt.Println()
 		fmt.Println("Usage:")
-		fmt.Printf("	mt-store-cat [flags] <normal|summary> id <metric-id>\n")
+		fmt.Printf("	mt-store-cat [flags] <normal|summary> id <metric-id> <ttl>\n")
```

Comment:
no i meant just the ttl pattern like `5h`, `3d`, etc. they could just reuse those values and we could apply the same parsing.